pandas/tests/frame/test_timeseries.py

```diff
@@ -21,6 +21,21 @@
 import pandas.tseries.offsets as offsets
 
 
+def test_no_idea_how_to_name_this_or_where_to_place_this():
```

Comment:
```suggestion
def test_timestamp_and_time_dtype_raises():
```

Reply:
the best place for this would be in tests.arithmetic.test_datetime64, but it will need to be re-written to use the box_with_array fixture